tests/sentry/relay/config/test_metric_extraction.py

```diff
@@ -452,32 +452,6 @@ def test_get_metric_extraction_config_multiple_widgets_duplicated(default_projec
         }
 
 
-@django_db_all
-def test_get_metric_extraction_config_alert_and_widget_deduplicated(
```

Comment:
I could have pushed this test a little more to make sure it tests for equality of specs but the one I add down below is simpler.